examples/tabular/example_quantile_regression.py

```diff
@@ -1,24 +1,38 @@
 """ Example script for quantile regression with tabular data, demonstrating simple use-case """
-
+import numpy as np
 from autogluon.tabular import TabularDataset, TabularPredictor
 
 # Training time:
 train_data = TabularDataset('https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')  # can be local CSV file as well, returns Pandas DataFrame
-train_data = train_data.head(500)  # subsample for faster demo
+train_data = train_data.head(1000)  # subsample for faster demo
 print(train_data.head())
 
 label = 'age'  # specifies which column do we want to predict
 save_path = 'ag_models/'  # where to save trained models
-quantiles_topredict = [0.1, 0.5, 0.9]  # which quantiles of numeric label-variable we want to predict
+quantile_levels = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]  # which quantiles of numeric label-variable we want to predict
```

Comment:
overall this example took a while for me to run. It would be better if it can run faster while still producing meaningful results. I get that we need enough training data for conformalization to work, but perhaps we do not need to estimate so many quantiles?

Reply:
Yes, that is right, more training data could improve the conformalization, but not nessecarily many quantiles.